package.json

```diff
@@ -45,6 +45,18 @@
       "<rootDir>/bootstrap/"
     ]
   },
+  "nodemonConfig": {
+    "execMap": {
+      "js": "babel-node -r dotenv/config"
+    },
```

Comment:
This just tells `nodemon` to use this command if given a `.js` file (so we can clean up the script above).